server/routes/testrpc.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const router = express.Router();
+const exec = require('child_process').exec;
+const config = require('../config.js');
+
+/* Port number for currently running "testrpc" */
+let port = '';
```

Comment:
only single test-rpc instance supported by this mechanism. Adding support for multiple instances should not be that much work, so the port can be used as a key, and process reference as a value in a map.